install_config/topics/ocp_osp_dns.adoc

```diff
@@ -0,0 +1,48 @@
+==== Setting up DNS for {product-title}
+
+DNS service is an important component in the {product-title} environment. Regardless of 
+the provider of DNS, an organization is required to have certain records in 
+place to serve the various {product-title} components.
+
+[WARNING] 
+====
+Using `/etc/hosts` is not valid, a proper DNS service must exist.
+====
+
+Using the key secret of the DNS, you can provide the information
+to the OpenShift Ansible Installer and it will automatically add A records for
+the target instances and the various {product-title} components. This process setup
+is described later when configuring the OpenShift Ansible Installer. 
+
+Access to a DNS server is expected. You can use
+link:https://access.redhat.com/labsinfo/dnshelper[Red Hat Labs DNS Helper] for 
+assistance with access.
+
+===== Application DNS
+
+Applications served by OpenShift are accessible by the router on  ports 80/TCP
+and 443/TCP.  The router uses a _wildcard_ record to map all host names under a
+specific sub domain to the same IP address without requiring a separate record
+for each name.
+
+This allows {product-title} to add applications with arbitrary names as
+long as they are under that sub domain.
+
+For example, a wildcard record for `*.apps.example.com` causes DNS name lookups
+for `tax.apps.example.com` and `home-goods.apps.example.com`
+to both return the same IP address: `10.19.x.y`. All 
+traffic is forwarded to the OpenShift Routers. The Routers examine the HTTP
+headers of the queries and forward them to the correct destination.
+
+With a load-balancer such as Octavia, host address of 10.19.x.y, the wildcard
+DNS record can be added as follows:
+
+.Load Balancer DNS records
+|===
+|IP Address |Hostname |Purpose
+
+|10.19.x.y
+|`*.apps.example.com`
+|User access to application web services
+
+|===
```

Comment:
It is good to me.
But this section describe the basic infrastructure of Openstack, and it is better to ask expert in Openstack team to review.